packages/eslint-config-compass/.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
this doesn't need something centralised right? i guess the idea is that it will be forever just the default, or?

Reply:
Yeah, exactly!